x-pack/plugins/actions/server/builtin_action_types/servicenow/service_sir.ts

```diff
@@ -29,22 +17,19 @@ const getAddObservableToIncidentURL = (url: string, incidentID: string) =>
 const getBulkAddObservableToIncidentURL = (url: string, incidentID: string) =>
   `${url}/api/x_elas2_sir_int/elastic_api/incident/${incidentID}/observables/bulk`;
 
-export const createExternalServiceSIR: ServiceFactory<ExternalServiceSIR> = (
-  credentials: ExternalServiceCredentials,
-  logger: Logger,
-  configurationUtilities: ActionsConfigurationUtilities,
-  serviceConfig: SNProductsConfigValue
-): ExternalServiceSIR => {
-  const snService = createExternalService(
+export const createExternalServiceSIR: ServiceFactory<ExternalServiceSIR> = ({
+  credentials,
+  logger,
+  configurationUtilities,
+  serviceConfig,
+  axiosInstance,
+}): ExternalServiceSIR => {
+  const snService = createExternalService({
     credentials,
     logger,
     configurationUtilities,
-    serviceConfig
-  );
-
-  const { username, password } = credentials.secrets as ServiceNowSecretConfigurationType;
```

Comment:
I saw that an axios instance was created when calling createExternalService and then another one was created here. I refactored the axios instance creation into create_service_wrapper and updated these service factory functions to use the passed-in axios instance.